build.washingtonpost.com/components/Layout/Components/Sidebar.js

```diff
@@ -212,7 +212,7 @@ export default function Sidebar({ navigation, setMobileMenu }) {
                     }}
                   >
                     <div>{item.frontMatter.data.title}</div>
-                    <Change type="Draft">Draft</Change>
+                    <Change type="Alpha">Alpha</Change>
```

Comment:
```suggestion
                    <Change type="Alpha" />
```
Maybe we could abstract away the children.